### ISSUE TITLE
Money not showing

### ISSUE THREAD

**Issue by PoisonXS** [bitpay/copay#6997]
On selly.gg I had an order which was successful.
I used the address that I have on my current wallet.
But when I look to see my balance it says no transactions and no balance.
I copied the exact same address.
My earlier post was wrong as I was looking at the wrong address.
I can post all transaction details.

**Comment by JDonadio** [comment]
Do you have the transaction id?

**Comment by PoisonXS** [comment]
Ah I just realised something.
When I first signed up to copay there was already an address for me, so I copied that onto my selly.gg settings. But I accidentally generated a new one and didn't update it. 
Here is the transaction information https://blockchain.info/address/3DawgasoDoaVubYvZjkWTHrfdwt4hvMKN1.

Can you please help me?

**Comment by JDonadio** [comment]
@PoisonXS sorry but I don't see the issue. Copay generate a new address once a transaction arrives to your wallet. Check this out: https://github.com/bitpay/copay/wiki/COPAY---FAQ#why-my-address-keeps-changing

If you need a "one single address wallet", you can create a new one using this option to prevent that behavior: 

<img width="355" alt="copay_-_copay_bitcoin_wallet" src="https://user-images.githubusercontent.com/13851807/32170328-e1f14034-bd52-11e7-92c8-e7927a8f19ca.png">

**Comment by PoisonXS** [comment]
But why don't I have any balance?
The transaction said it was successful, I copied the address from Copay.
Please help me as this isn't fair

**Comment by JDonadio** [comment]
I need the transaction id and also your wallet id to check your wallet status. The wallet id is located in settings > wallet > more opts > wallet information.

**Comment by PoisonXS** [comment]
Wallet Id: 38fb43c2-0749-40b8-9d9b-8fa6a8b2a373

I don't understand about the transaction id.

**Comment by JDonadio** [comment]
This is your wallet status:

```* Balance
  Total: 0.00000000 btc (0.00000000 btc locked)
  Confirmed: 0.00000000 btc (0.00000000 btc locked)
  Available: 0.00000000 btc (0.00000000 btc confirmed / 0.00000000 btc unconfirmed)

* No pending tx proposals.

* Main addresses (not including change addresses)
  [Tue Oct 24 2017 22:59:37 GMT-0300] 17UtbNiVXdp99t3fRxqM2c1czajoxabSZu (m/0/0)
  [Tue Oct 24 2017 23:09:23 GMT-0300] 1PWWnv3XXy6dYbjJ1ESAh4A828LEpkymBY (m/0/1)
  [Wed Oct 25 2017 11:50:16 GMT-0300] 1KkV3bUtRLWLcnxeVVh8jnS7B6S95dSCEr (m/0/2)

* No tx history.
```

**Comment by JDonadio** [comment]
You never receive any transaction on that wallet. Contact to the sender and verify your addresses.
